CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Feature
+
+- Includes prop on product summary image to handle image label render on mobile
```

Comment:
Please update the CHANGELOG according to https://github.com/vtex-apps/store-discussion/blob/master/CONTRIBUTING.md.